packages/frontend-shared/src/composables/examples/UseCohortsExample.cy.tsx

```diff
@@ -0,0 +1,21 @@
+import { WEIGHTED } from '../../utils/weightedChoice'
```

Comment:
What does this test prove? Since it comes down to randomness, all you can assert on is that `Notification Title` is present which doesn't add much value, especially since you've hooked up `useCohort` in other CT tests.